component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateUtil.java

```diff
@@ -373,4 +379,91 @@ private static boolean isUserExists(String userName, AuthenticationContext authe
         }
     }
 
-}
\ No newline at end of file
+    /**
+     * Check whether user account is locked or not.
+     *
+     * @param user Authenticated user.
+     * @return boolean account locked or not.
+     * @throws AccountLockServiceException
```

Comment:
Complete the comment with exception description.

Reply:
Shall we fix in other places as well?

Reply:
Fixed with ed2968b2c84d173790ff71b500a0268434981d01